MobileWallet/Screens/Debug/HomeViewControllerDebugExtension.swift

```diff
@@ -46,9 +46,9 @@ extension HomeViewController {
         present(logsVC, animated: true, completion: nil)
     }
 
-    private func generateTestData() {
+    private func generateTestData(filepath: String) {
```

Comment:
Why does this need the file path? `generateTestData` doesn't actually have anything to do with the unit tests, just a debug option when running the app. DB path is set as a constant in the lib so the UI classes should never have to worry about that path.

Reply:
The method changed in the header

```C
bool wallet_test_generate_data(struct TariWallet *wallet, char *datastore_path,int* error_out);
```

Reply:
That's fine then the but to avoid improper use of method, `*datastore_path` can be the `databasePath` from TariLib instead of being manually overridden here